governance/ACCESS_POLICY.md

```diff
@@ -0,0 +1,14 @@
+# Account Access Policy
+
+## Zoom
+
+Ask the Community Committee Chairperson for access. The chairpersons of the TSC and Community Committee have access to grant their membership YouTube streaming.
+
```

Comment:
maybe
grant access to projected members so they can start a YouTube stream.

Reply:
I think in the end we agreed that requests would go to Brian who can share through lastpass?

Reply:
AFAIK that's not currently set up. Any issue with merging it as-is and PRing the change once it's implemented?

Reply:
@brianwarner are you ready to start taking requests/sharing?

Reply:
Guess what I got yesterday?  If you guessed Node Zoom credentials, you'd be right ;-)

I can share these from LastPass now.  Just let me know who is approved under the access policy.

Reply:
@bnb I think since Brian is ready we should update to reflect he's the once to reach out to.

Reply:
@brianwarner what is the process you'd like to follow for people to reach out to you?

Reply:
@bnb Email is best.  I'd suggest it be operations@openjsf.org (which forwards to me, or if I'm on vacation, I can delegate it to someone else).